src/main/java/edu/kis/powp/jobs2d/drivers/command/ComplexCommand.java

```diff
@@ -0,0 +1,25 @@
+package edu.kis.powp.jobs2d.drivers.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.List;
+
+public class ComplexCommand implements DriverCommand {
+    List<DriverCommand> commandList;
```

Comment:
should be private!

Reply:
added private access identifier to complexCommand list.